common/api-review/util.api.md

```diff
@@ -487,9 +487,6 @@ export interface Subscribe<T> {
 // @public (undocumented)
 export type Unsubscribe = () => void;
 
-// @public
-export function updateEmulatorBanner(name: string, isRunningEmulator: boolean): void;
```

Comment:
I'm guessing the idea is to remove the call in all the product packages, but not remove the implementation from util, so you undid the util file rollback but missed this file. In that case, revert this file to fix the docgen CI check.

Reply:
Done